Cargo.toml

```diff
@@ -14,4 +14,6 @@ homepage = "http://www.maidsafe.net"
 rand = "*"
 rust-crypto ="*"
 tempdir = "*"
+docopt_macros = "*"
+rustc-serialize = "*"
```

Comment:
rustc-serialize is a dependecy of rust-crypto, we don't use it ourself directly - I think